src/assets/TokensController.test.ts

```diff
@@ -16,6 +17,7 @@ describe('TokensController', () => {
   let preferences: PreferencesController;
   let network: NetworkController;
 
+  let instEthProvStub: any;
```

Comment:
Does Sinon give us a type we can use? I feel like we should avoid the use of `any` as much as possible, even in tests.